### DISCUSSION TITLE
Do Skywalking-Java-Agent and Skywalking-servers use grpc protocol to communicate support compression？

### DISCUSSION THREAD

**Discussion by GuiSong01** [apache/skywalking#10150] [Q&A] [answered]
If the Skywalking-Java-Agent and Skywalking-server are not deployed in the same region, compression reduces bandwidth costs

**Comment by wu-sheng** [comment, marked as answer]
That relies on gRPC. I don't think we have exposed options for that. Contributions are welcome.
